src/ekat/util/ekat_arch.cpp

```diff
@@ -32,7 +32,7 @@ std::string active_avx_string () {
 std::string ekat_config_string () {
   std::stringstream ss;
   ss << "ExecSpace name: " << DefaultDevice::execution_space::name() << "\n";
-  ss << "ExecSpace initialized: " << (DefaultDevice::execution_space::impl_is_initialized() ? "yes" : "no") << "\n";
+  ss << "ExecSpace initialized: " << (DefaultDevice::execution_space::is_initialized() ? "yes" : "no") << "\n";
```

Comment:
As I explained in #20, this is not a bug, but a configuration issue with Kokkos. This mod should be reverted, or, at least, made conditional:

```
#ifdef KOKKOS_ENABLE_DEPRECATED_CODE
is_initialized() version here
#else
impl_is_initialized() version here.
#endif
```